common/api/ecschema-metadata.api.md

```diff
@@ -795,16 +795,14 @@ export class Format extends SchemaItem {
     get uomSeparator(): string;
 }
 
-// @beta
+// @public @preview
```

Comment:
Be careful once this goes public preview you cannot add new required parameters.

I cannot think of another thing we need to add but I would old off until we've gotten some feedback from someone designing and using their own format set.

Reply:
I've changed the interface back to beta